docs/libraries.md

```diff
@@ -0,0 +1,77 @@
+---
+id: libraries
+title: Using Libraries
+author: Brent Vatne
+authorURL: https://twitter.com/notbrent
+description: This guide introduces React Native developers to finding, installing, and using third-party libraries in their apps.
+---
+
+React Native provides a set of built-in [Core Components and APIs](./components-and-apis) ready to use in your app. You're not limited to the components and APIs bundled with React Native. React Native has a community of thousands of developers. If the Core Components and APIs don't have what you are looking for, you may be able to find and install a library from the community to add the functionality to your app.
+
+## Selecting a Package Manager
+
+React Native libraries are typically installed from the [npm registry](https://www.npmjs.com/) using a Node.js package manager such as [npm CLI](https://docs.npmjs.com/cli/npm) or [Yarn Classic](https://classic.yarnpkg.com/).
+
+If you have Node.js installed on your computer then you already have the npm CLI installed. Some developers prefer to use Yarn Classic for slightly faster install times and additional advanced features like Workspaces. Both tools work great with React Native. We will assume npm for the rest of this guide for simplicity of explanation.
+
+> 💡 The terms "library" and "package" are used interchangably in the JavaScript community.
+
+## Installing a Library
+
+To install a library in your project, navigate to your project directory in your terminal and run `npm install <name-of-the-library>`. Let's try this with `react-native-webview`:
+
+```bash
+npm install react-native-webview
+```
+
+The library that we installed includes native code, and we need to link to our app before we use it.
```

Comment:
i wish we had an easy way for developers to determine "does this library include native code and require me to rebuild the native project?"